### DISCUSSION TITLE
Turn on gopro from gpio, record for 200 seconds, shut down

### DISCUSSION THREAD

**Discussion by twaddellberkeley** [gopro/labs#840] [Q&A]
Hello,
I am looking to do as the title describes, turn on the gopro once a power signal is received, record for 20 seconds, and then shutdown,
I have successfully used the command !MBOOT="!Lboot"SAVEboot=!S20E to do everything except stop recording after 20 seconds, is there any reason this wouldnt work?

**Comment by dnewman-gpsw** [comment]
There is typo in that command.  This is the full command needed:

```!MWAKE=2!MBOOT="!Lboot"!SAVEboot=!S20E```
